src/Development/Shake/Command.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE FlexibleInstances, TypeSynonymInstances, TypeOperators, ScopedTypeVariables, NamedFieldPuns #-}
+{-# LANGUAGE FlexibleInstances, GADTs, TypeSynonymInstances, TypeOperators, ScopedTypeVariables, MultiParamTypeClasses, NamedFieldPuns #-}
```

Comment:
I don't see a use of either GADTs or MPTC in the code below - are they necessary?